layout/theme.liquid

```diff
@@ -17,16 +17,14 @@
 
     {% include 'page-metadata' %}
 
-    {%- if template == "index" -%}
-      <script>
-        // Loading the IFrame Player API code asynchronously.
-        const tag = document.createElement('script');
-        tag.src = "https://www.youtube.com/iframe_api";
-
-        const firstScriptTag = document.getElementsByTagName('script')[0];
-        firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
-      </script>
-    {%- endif -%}
+    <script>
+      // Loading the IFrame Player API code asynchronously.
+      const tag = document.createElement('script');
+      tag.src = "https://www.youtube.com/iframe_api";
+
+      const firstScriptTag = document.getElementsByTagName('script')[0];
+      firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+    </script>
```

Comment:
Won't the `<script src="..." async></script>` work here?

Reply:
Unfortunately, this approach doesn't work but I found a new solution for this issue. I've already implemented and checked it in the Chameleon theme, seems it works properly on all devices I have. 
So I will share this solution with Impact theme but when I will work with Hero section.

Reply:
Okay, then I think you can at least simplify the approach with something like the following
```liquid
<script>
  const tag = document.createElement('script');

  tag.src = 'https://www.youtube.cim/iframe_api';

  document.body.appendChild(tag);
</script>
```

Reply:
FIxed